### ISSUE TITLE
[rt.cpan.org #71906] [Patch] Missing 'E' in front of POD entity in perlfaq7.pod

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11715]
Migrated from [rt.perl.org#102084](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=102084) (status was 'rejected')

Searchable as RT102084$

**Comment by p5pRT** [comment]
### From @trwyant

#### Created by @trwyant



Thank you very much for Perl 5\.15\.4\.

The perlfaq7 document as released in 5\.15\.4 contains the line

 =head2 Why doesn't "my\($foo\) = E\<lt>$fh\<gt>;" work right?

Was this supposed to be

 =head2 Why doesn't "my\($foo\) = E\<lt>$fhE\<gt>;" work right?

I\.e\. is there a missing 'E' in front of the '\<gt>' after '$fh'?
 
A universal diff of the proposed change is attached\.

Thank you very much for making Perl great\, and for reading to the bottom
of this nit\-pick\.

Tom Wyant \(wyant@&#8203;cpan\.org\, irrespective of actual sending address\)

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=low

Site configuration information for perl 5.15.4:

Configured by tom at Fri Oct 21 09:38:53 EDT 2011.

Summary of my perl5 (revision 5 version 15 subversion 4) configuration:
   
  Platform:
    osname=darwin, osvers=9.8.0, archname=darwin-2level
    uname='darwin wyants-white-too.local 9.8.0 darwin kernel version 9.8.0: wed jul 15 16:55:01 pdt 2009; root:xnu-1228.15.4~1release_i386 i386 '
    config_args='-Dprefix=/usr/local/perl/5.15.4'
    hint=recommended, useposix=true, d_sigaction=define
    useithreads=undef, usemultiplicity=undef
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=undef, use64bitall=undef, uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -I/opt/local/include',
    optimize='-O3',
    cppflags='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -I/opt/local/include'
    ccversion='', gccversion='4.0.1 (Apple Inc. build 5484)', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='env MACOSX_DEPLOYMENT_TARGET=10.3 cc', ldflags =' -fstack-protector -L/usr/local/lib -L/opt/local/lib'
    libpth=/usr/local/lib /opt/local/lib /usr/lib
    libs=-lgdbm -ldbm -ldl -lm -lutil -lc
    perllibs=-ldl -lm -lutil -lc
    libc=, so=dylib, useshrplib=false, libperl=libperl.a
    gnulibc_version=''
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=bundle, d_dlsymun=undef, ccdlflags=' '
    cccdlflags=' ', lddlflags=' -bundle -undefined dynamic_lookup -L/usr/local/lib -L/opt/local/lib -fstack-protector'

Locally applied patches:
    


@INC for perl 5.15.4:
    /usr/local/perl/5.15.4/lib/site_perl/5.15.4/darwin-2level
    /usr/local/perl/5.15.4/lib/site_perl/5.15.4
    /usr/local/perl/5.15.4/lib/5.15.4/darwin-2level
    /usr/local/perl/5.15.4/lib/5.15.4
    .


Environment for perl 5.15.4:
    DYLD_LIBRARY_PATH (unset)
    HOME=/Users/tom
    LANG=en_US.UTF-8
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/usr/local/perl/5.15.4/bin:/opt/local/bin:/opt/local/sbin:/ImageMagick-6.4.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/X11/bin:/Users/tom/Code/Tools
    PERLPKGOPT=-noppm
    PERL_BADLANG (unset)
    SHELL=/bin/bash
```
</details>

**Comment by p5pRT** [comment]
### From @trwyant

<details><summary>perlfaq7.patch</summary>

```diff
--- perlfaq7.old	2011-10-22 11:25:11.000000000 -0400
+++ perlfaq7.pod	2011-10-24 18:57:51.000000000 -0400
@@ -604,7 +604,7 @@
 are effectively shallowly bound. Consider this just one more reason
 not to use them. See the answer to L<"What's a closure?">.
 
-=head2 Why doesn't "my($foo) = E<lt>$fh<gt>;" work right?
+=head2 Why doesn't "my($foo) = E<lt>$fhE<gt>;" work right?
 
 C<my()> and C<local()> give list context to the right hand side
 of C<=>. The <$fh> read operation, like so many of Perl's

```
</details>

**Comment by p5pRT** [comment]
### From @cpansprout

perlfaq is no longer maintained as part of the perl core\.  It’s still
included\, but CPAN is now upstream\, and it has its own CPAN
distribution\.  I’m forwarding this to the CPAN queue\.  This change is so
recent that yours will be the first bug report there\.

On Mon Oct 24 16&#8203;:14&#8203;:23 2011\, wyant wrote&#8203;:
> This is a bug report for perl from wyant@&#8203;cpan\.org\,
> generated with the help of perlbug 1\.39 running under perl 5\.15\.4\.
> 
> 
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> \[Please describe your issue here\]
> 
> Thank you very much for Perl 5\.15\.4\.
> 
> The perlfaq7 document as released in 5\.15\.4 contains the line
> 
>  =head2 Why doesn't "my\($foo\) = E\<lt>$fh\<gt>;" work right?
> 
> Was this supposed to be
> 
>  =head2 Why doesn't "my\($foo\) = E\<lt>$fhE\<gt>;" work right?
> 
> I\.e\. is there a missing 'E' in front of the '\<gt>' after '$fh'?
> 
> A universal diff of the proposed change is attached\.
> 
> Thank you very much for making Perl great\, and for reading to the
> bottom
> of this nit\-pick\.
> 
> Tom Wyant \(wyant@&#8203;cpan\.org\, irrespective of actual sending address\)
> \[Please do not change anything below this line\]
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> \-\-\-
> Flags&#8203;:
>     category=docs
>     severity=low
> \-\-\-
> Site configuration information for perl 5\.15\.4&#8203;:
> 
> Configured by tom at Fri Oct 21 09&#8203;:38&#8203;:53 EDT 2011\.
> 
> Summary of my perl5 \(revision 5 version 15 subversion 4\)
> configuration&#8203;:
> 
>   Platform&#8203;:
>     osname=darwin\, osvers=9\.8\.0\, archname=darwin\-2level
>     uname='darwin wyants\-white\-too\.local 9\.8\.0 darwin kernel version
> 9\.8\.0&#8203;: wed jul 15 16&#8203;:55&#8203;:01 pdt 2009; root&#8203;:xnu\-1228\.15\.4~1release\_i386
> i386 '
>     config\_args='\-Dprefix=/usr/local/perl/5\.15\.4'
>     hint=recommended\, useposix=true\, d\_sigaction=define
>     useithreads=undef\, usemultiplicity=undef
>     useperlio=define\, d\_sfio=undef\, uselargefiles=define\,
> usesocks=undef
>     use64bitint=undef\, use64bitall=undef\, uselongdouble=undef
>     usemymalloc=n\, bincompat5005=undef
>   Compiler&#8203;:
>     cc='cc'\, ccflags ='\-fno\-common \-DPERL\_DARWIN \-fno\-strict\-aliasing
> \-pipe \-fstack\-protector \-I/usr/local/include \-I/opt/local/include'\,
>     optimize='\-O3'\,
>     cppflags='\-fno\-common \-DPERL\_DARWIN \-fno\-strict\-aliasing \-pipe
> \-fstack\-protector \-I/usr/local/include \-I/opt/local/include'
>     ccversion=''\, gccversion='4\.0\.1 \(Apple Inc\. build 5484\)'\,
> gccosandvers=''
>     intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=1234
>     d\_longlong=define\, longlongsize=8\, d\_longdbl=define\,
> longdblsize=16
>     ivtype='long'\, ivsize=4\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\,
> lseeksize=8
>     alignbytes=8\, prototype=define
>   Linker and Libraries&#8203;:
>     ld='env MACOSX\_DEPLOYMENT\_TARGET=10\.3 cc'\, ldflags ='
> \-fstack\-protector \-L/usr/local/lib \-L/opt/local/lib'
>     libpth=/usr/local/lib /opt/local/lib /usr/lib
>     libs=\-lgdbm \-ldbm \-ldl \-lm \-lutil \-lc
>     perllibs=\-ldl \-lm \-lutil \-lc
>     libc=\, so=dylib\, useshrplib=false\, libperl=libperl\.a
>     gnulibc\_version=''
>   Dynamic Linking&#8203;:
>     dlsrc=dl\_dlopen\.xs\, dlext=bundle\, d\_dlsymun=undef\, ccdlflags=' '
>     cccdlflags=' '\, lddlflags=' \-bundle \-undefined dynamic\_lookup
> \-L/usr/local/lib \-L/opt/local/lib \-fstack\-protector'
> 
> Locally applied patches&#8203;:
> 
> 
> \-\-\-
> @&#8203;INC for perl 5\.15\.4&#8203;:
>     /usr/local/perl/5\.15\.4/lib/site\_perl/5\.15\.4/darwin\-2level
>     /usr/local/perl/5\.15\.4/lib/site\_perl/5\.15\.4
>     /usr/local/perl/5\.15\.4/lib/5\.15\.4/darwin\-2level
>     /usr/local/perl/5\.15\.4/lib/5\.15\.4
>     \.
> 
> \-\-\-
> Environment for perl 5\.15\.4&#8203;:
>     DYLD\_LIBRARY\_PATH \(unset\)
>     HOME=/Users/tom
>     LANG=en\_US\.UTF\-8
>     LANGUAGE \(unset\)
>     LD\_LIBRARY\_PATH \(unset\)
>     LOGDIR \(unset\)
>    
PATH=/usr/local/perl/5\.15\.4/bin&#8203;:/opt/local/bin&#8203;:/opt/local/sbin&#8203;:/ImageMagick\-
>
6\.4\.1/bin&#8203;:/usr/local/bin&#8203;:/usr/bin&#8203;:/bin&#8203;:/usr/sbin&#8203;:/sbin&#8203;:/usr/X11/bin&#8203;:/Users/tom/Code/Tools
>     PERLPKGOPT=\-noppm
>     PERL\_BADLANG \(unset\)
>     SHELL=/bin/bash

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@cpansprout - Status changed from 'open' to 'rejected'

**Comment by p5pRT** [comment]
### From bug-perlfaq@rt.cpan.org

\<URL&#8203;: https://rt.cpan.org/Ticket/Display.html?id=71906 >

On Mon Oct 24 19&#8203;:37&#8203;:10 2011\, perlbug\-followup@&#8203;perl\.org wrote&#8203;:
> perlfaq is no longer maintained as part of the perl core\.  It’s still
> included\, but CPAN is now upstream\, and it has its own CPAN
> distribution\.  I’m forwarding this to the CPAN queue\.  This change is
> so
> recent that yours will be the first bug report there\.
> 
> On Mon Oct 24 16&#8203;:14&#8203;:23 2011\, wyant wrote&#8203;:
> > This is a bug report for perl from wyant@&#8203;cpan\.org\,
> > generated with the help of perlbug 1\.39 running under perl 5\.15\.4\.
> >
> >
> > \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> > \[Please describe your issue here\]
> >
> > Thank you very much for Perl 5\.15\.4\.
> >
> > The perlfaq7 document as released in 5\.15\.4 contains the line
> >
> >  =head2 Why doesn't "my\($foo\) = E\<lt>$fh\<gt>;" work right?
> >
> > Was this supposed to be
> >
> >  =head2 Why doesn't "my\($foo\) = E\<lt>$fhE\<gt>;" work right?

Fixed in git&#8203;:

https://github.com/perl-doc-
cats/perlfaq/commit/071fb5bee75b55330be91b095078d8fc5af5b98e

Thanks

Leo
